### PR TITLE
Browser Monitoring Edit

### PR DESCRIPTION
**Pull request by alai97**
## Motivation

#rum-docs Slack

## Changes

Adds copyedits to the Browser Monitoring doc index page, moves some content into a guide.

## Preview

https://docs-staging.datadoghq.com/alai97/rum-browser-and-browser-premium-sampling-guide/real_user_monitoring/browser/ (the link to the guide page does not work here yet).

https://docs-staging.datadoghq.com/alai97/rum-browser-and-browser-premium-sampling-guide/real_user_monitoring/guide/sampling-browser-and-browser-premium

## Testing

- [ ] Local
- [ ] Staging
- [ ] Unit
- [ ] End to end

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

File: packages/rum/README.md
Comment (alai97) [inline packages/rum/README.md:453]:
```suggestion
{{< img src="real_user_monitoring/browser/example-initialization-snippet.mp4" alt="Example initialization snippet for a browser application with custom percentages" video="true" width="100%" >}}
```

File: packages/rum/README.md
Comment (alai97) [inline packages/rum/README.md:395]:
```suggestion
{{< img src="real_user_monitoring/browser/browser-initialization-snippet.png" alt="Default initialization snippet for a browser application" style="width:100%;" >}}
```

**Review by maycmlee — APPROVED** [review]

File: packages/rum/README.md
Comment (alai97) [inline packages/rum/README.md:492]:
```suggestion
You can explore the following attributes:
```

**Review by liywjl — APPROVED** [review]

File: packages/rum/README.md
Comment (BenoitZugmeyer) [inline packages/rum/README.md:453]: **🔨 warning:** ‏This won't render well in Github. AFAIK, you can't add a video to a GitHub README... maybe add just a link to it?

**Review by BenoitZugmeyer — CHANGES_REQUESTED** [review]

File: packages/rum/README.md
Comment (BenoitZugmeyer) [reply packages/rum/README.md:453]: It looks like a lot of documentation edits are about clarifying the sample rate options. Maybe having a dedicated page related to sample rate configuration could be an interesting option, so we don't have github-related limitations.

Comment (alai97) [reply packages/rum/README.md:453]:
```suggestion
```

File: packages/rum/README.md
Comment (BenoitZugmeyer) [inline packages/rum/README.md:5]: **❓ question:** ‏Why changing the name? We have two "Browser SDK", Logs and RUM. This doc is about RUM, so IMO it should be `RUM Browser SDK` (or eventually `Datadog RUM Browser SDK`)

File: packages/rum/README.md
Comment (BenoitZugmeyer) [inline packages/rum/README.md:377]: **💬 suggestion:** ‏Move this whole section into the new sampling guide. This way, everything is documented at the same place. We can just link to it directly.

Comment (jamiemilstein) [reply packages/rum/README.md:377]: i like that idea

File: packages/rum/README.md
Comment (jamiemilstein) [inline packages/rum/README.md:337]: "you using" feels like a typo/gramatically incorrect. you're?

**Comment by jamiemilstein** [comment]
On the guides page, I don't think [this title](https://a.cl.ly/bLu51WlA) makes sense for it. It's more about the setup

File: packages/rum/README.md
Comment (alai97) [reply packages/rum/README.md:337]:
```suggestion
Options that must have matching configuration when you are using the Datadog Browser SDK for logs:
```

**Comment by alai97** [comment]
@BenoitZugmeyer feel free to merge!

File: packages/rum/README.md
Comment (bcaudan) [inline packages/rum/README.md:475]: **❗ warning:** ‏this doc must be merged before merging this PR

**Comment by BenoitZugmeyer** [comment]
I moved the Browser SDK renaming to a separate PR #1707 as I want to submit it to PM approval beforehand.

File: packages/rum/README.md
Comment (alai97) [reply packages/rum/README.md:475]: My linked PR (https://github.com/DataDog/documentation/pull/15038) has been merged!

**Review by BenoitZugmeyer — APPROVED** [review]

**Pull request merged**
